guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,55 @@
 
 In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
 
-You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
+Dependency solving is relevant only in limited contexts.
+In the most common use cases, such as adding a security errata to a content view with incremental update, dependency solving is likely unnecessary and can cause significant delays to Content View publication.
 
-Note that resolving package dependencies can cause significant delays to Content View promotion.
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+The package dependency resolution feature does not take into account which packages are installed on your clients.
+Dependency resolution considers only packages within the repositories of the content view.
```

Comment:
Let's keep it to two names for the same thing - "the package dependency resolution feature" or "dependency solving" for short.

```suggestion
Dependency solving considers only packages within the repositories of the content view.
```